temporalio/common/retry_policy.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import Optional
+
+import temporalio.api.common.v1
+
+
+@dataclass
+class RetryPolicy:
+    initial_interval: timedelta
+    randomization_factor: float
+    multiplier: float
+    max_interval: timedelta
+    max_elapsed_time: Optional[timedelta]
+    max_retries: int
+
+    def apply_to_proto(self, proto: temporalio.api.common.v1.RetryPolicy) -> None:
```

Comment:
I'd prefer if this was a static function instead of a method on this class because it's a public facing class and users should never have to call this directly.

Reply:
I'm saying static as in not attached to a class and not exported but since we offer the gRPC API I agree this can be left on the class.